msdsn/conn_str.go

```diff
@@ -129,6 +130,37 @@ type Config struct {
 	ColumnEncryption bool
 }
 
+func readDERFile(filename string) ([]byte, error) {
```

Comment:
>File(filenam [](http://example.com/codeflow?start=12&length=12)

add a unit test. Hard code some bytes from some unused certificate, write them to a file, then call this method to validate.

Reply:
Added unit test